apps/studio/components/interfaces/SQLEditor/UtilityPanel/ChartConfig.tsx

```diff
@@ -168,6 +188,28 @@ export function ChartConfig({ results = { rows: [] }, config, onConfigChange }:
                 Cumulative
               </Label_Shadcn_>
             </div>
+            <div className="flex items-center">
+              <Checkbox_Shadcn_
+                id="showLabels"
+                name="showLabels"
+                checked={config.showLabels}
+                onClick={() => onConfigChange({ ...config, showLabels: !config.showLabels })}
+              />
+              <Label_Shadcn_ className="text-foreground-light p-1.5" htmlFor="showLabels">
```

Comment:
It's possible these labels won't be interactive unless you wrap a <form> around the whole thing. a weird Radix thing, it only inserts a hidden `<input />` when inside a `<form />`

Reply:
seems to work fine (clicking either the label or the checkbox) but added the <form> justincase something changes in the future 🤔